### DISCUSSION TITLE
How do I get to settings.cfg for Terrarium Pi 3.9.9?

### DISCUSSION THREAD

**Discussion by Plantbrowser** [theyosh/TerrariumPI#555] [Q&A]
What the title says. I found the FAQ and explains what to do but I have no idea how to get to the file as a novice with little experience with Linux. Would the average be excluded in 4.0.0?

Thank you for reading!

**Comment by Plantbrowser** [comment]
I was able to find the file after coming across a post by blatogera. I saw his screen shot and figured it out. Got to the terrarium pi folder and used the command -ls

**Comment by Plantbrowser** [comment]
After changing the exclude average input. My dashboard no longer displays temperature and humidity information.
![dash board empty](https://user-images.githubusercontent.com/84762227/121790187-66840280-cba2-11eb-9fc5-a3a948120ba4.png)

**Reply by theyosh** [reply]
Hi,

I am not sure what you are trying to do with the settings.cfg file. You should not change that manually.

So I get the feeling that your current settings.cfg file is corrupt. So the easiest way right now, it to login with SSH and rename the settings.cfg file and restart. This should give you a clean setup again, and then everything should work.

- Login with ssh https://howchoo.com/g/mgi3mdnlnjq/how-to-log-in-to-a-raspberry-pi-via-ssh
- Go to the TerrariumPI folder: `cd /home/pi/TerrariumPI`
- Rename the settings file: `mv settings.cfg settings.cfg.old`
- Restart TerrariumPI: https://github.com/theyosh/TerrariumPI/wiki/FAQ#how-to-start-terrariumpi (use screen method)

Then my question is what you are trying to do. Could you explain it?

**Reply by Plantbrowser** [reply]
Hello! I believe I messed up a file when saving. I figured it would be a faster solution if I did a clean install of terrarium pi. I think a possible explanation is I excluded all the sensors therefore there was no "average" information. I did not any other way to manually exclude a sensor.
